app.js

```diff
@@ -745,6 +757,9 @@ d.run(() => {
 				 * @param {function} cb - Callback function
 				 */
 				function(scope, cb) {
+					if (parseInt(appConfig.wsWorkers) < 1) {
+						return cb();
+					}
```

Comment:
I would suggest to implement this logic in other way. 

For now we have following steps in `async.auto`

- config
- logger
- build
- lastCommit
- geensisBlock 
- schema
- network 
- sequence 
- balanceSequence
- swagger
- ed
- bus
- db
- cache
- webSocket 
- logic
- modules 
- ready 
- listenWebSocket
- listenHttp 

So during snapshotting what we really don't need is not to run steps `listenWebSocket` and `listenHttp`. 

If we dig deeper, we don't even want to initialize web server or socket cluster, but that's a way ahead of the thought. 

I would suggest just change in those two steps `listenWebSocket` and `listenHttp` and make that conditional  based on `snapshotting` not any other config attribute.

Reply:
I tried your approach and the application keeps trying to update peers and releasing broadcast queue. My approach is suitable for another purposes.

Anyways, I've turned off `listenHttp` as well if snapshot is requested or api was disabled.